shared/actions/chat/messages.js

```diff
@@ -45,6 +47,14 @@ function* deleteMessage(action: Constants.DeleteMessage): SagaGenerator<any, any
       }
     }
 
+    const routePathSelector = (state: TypedState) => state.routeTree.routeState
+    const routePath = yield select(routePathSelector)
+
+    const numBack = getPath(routePath).count() - 3
```

Comment:
I'm confused by what this is doing. Is the intent to move back to the conversation view, clearing out any child routes? Would `navigateTo([], [chatTab, conversationId])` fit the bill?